configs/setup_golden_image.py

```diff
@@ -22,11 +22,16 @@
 from tmms.utils import core_utils
 from tmms.utils import file_utils
 from tmms.utils import customize_node
+from tmms.utils import logging
 from configs.build_config import ManifestingConfiguration
 
+LOGGER = None
+VERBOSE = False
```

Comment:
Ummmmmmmmm.....not happy with this one.....

Reply:
If its about "globals", then we violated this rule in manifest_api.py looong time ago :)